h/static/scripts/widget-controller.js

```diff
@@ -381,6 +387,7 @@ module.exports = function WidgetController(
   };
 
   $scope.isLoading = isLoading;
+  annotationUI.selectTab(uiConstants.TAB_ANNOTATIONS);
   $scope.tabAnnotations = uiConstants.TAB_ANNOTATIONS;
```

Comment:
I don't think `tabAnnotations` and `tabNotes` are needed any more are they, since you just import the constants directly in the file where they are used.